requirements_dev.txt

```diff
@@ -11,7 +11,7 @@ sphinx_rtd_theme==0.2.4
 
 pytest==3.4.2
 pytest-runner==4.0
-
+numpy
```

Comment:
I already added this. Can be removed.

Reply:
It will be removed/ignored at rebase time :)